src/components/common/notification/displayNotification.tsx

```diff
@@ -17,9 +17,13 @@ export const displayNotification = (
     </div>
   );
 
+  // set autoclose depending on message length
+  const msgLength = message ? message.length : 0;
```

Comment:
In modern JS you can do this: `const msgLength = message?.length ?? 0;`. Not that I think you should change it, just an FYI :D 

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator#relationship_with_the_optional_chaining_operator_